test/acceptance/user.controller.acceptance.ts

```diff
@@ -12,11 +12,11 @@ import {setupApplication} from './helper';
 import {createRecommendationServer} from '../../recommender';
 import {Server} from 'http';
 import * as _ from 'lodash';
-import {promisify} from 'util';
-import {hash} from 'bcryptjs';
-import {getAccessTokenForUser} from '../../src/utils/user.authentication';
+import {
+  getAccessTokenForUser,
+  hashPassword,
+} from '../../src/utils/user.authentication';
```

Comment:
We should try to avoid references to `src`. They should be imported from the module itself (assuming these functions are exported).

Reply:
@raymondfeng Could you explain more about what "module" means here? Is it the same thing as "artifact" like models, datasources...?

Reply:
It's npm module. Something similar as `import {createRecommendationServer} from '../..';`, which points to the root of this app.